sam/src/main/java/org/htsjdk/sam/SamRecord.java

```diff
@@ -0,0 +1,7 @@
+package org.htsjdk.sam;
+
+import org.htsjdk.core.api.Record;
+
+public class SamRecord implements Record {
```

Comment:
Change to `RecordImpl`, as this placeholder will be removed instead of implement (the same as the `Record` interface).

Reply:
I changed it to `CramStubRecord` which I think gets the point across that's not intended to be a final thing.

Reply:
I think that you forgot to add/commit that change!